### ISSUE TITLE
IIIF image viewer for ES images

### ISSUE THREAD

**Issue by PietroLiuzzo** [SChAth/ScAthiop#427]
https://github.com/SChAth/ScAthiop/wiki/SER3

**Comment by PietroLiuzzo** [comment]
To use Pandora, there are actually very few requirements.  The images should be in TIFF format for persistence in the repository.  To use your existing metadata and annotations with the iiif presentation api, these can be implemented in your own manifest service, probably without too much effort since you already have them as linked data.

I would suggest making a complete dump of your JSON-LD serialization then loading that into a graph store.  You can then use sparql construct to output your annotations as a list into the expected client serialization.   Here is an example list query: https://github.com/pan-dora/manifest-service/blob/master/public/api/graphs/list.rq

See also http://iiif.io/api/presentation/2.1/#annotation-list

**Comment by PietroLiuzzo** [comment]
ES images should not be directly downloadable, from viewer or with buttons.

**Comment by PietroLiuzzo** [comment]
#449

**Comment by PietroLiuzzo** [comment]
one test manuscript is now in the new 100 GB hard disk.

- test file is BMQ-003 (ESbmq003)
- files have been copied from UHHDisk 5 to local machine
- files have been uploaded to betamasaheft server on /home/liuzzo and then sudo mv to /media/add/images
- folder for institution is BMQ (should be the institution id...) and the progressive number is a subfolder. files are JPG. 
- files have been converted following  http://iipimage.sourceforge.net/documentation/images/
using the ad hoc script jpg2tif.sh in /media/add/images with BMQ/003 as argument. This takes some time.
- JPG files have been removed so that only .tif are in the folder.
- the folder size is 1.4 G. This is a MINIMAL improvement compared to the original 1.6G of the BMQM-003 User folder from the server containing NEF, JPG and resized JPG.

**Comment by PietroLiuzzo** [comment]
#452

**Comment by PietroLiuzzo** [comment]
we now have a new version of iip image server, 1.0 installed from git 
https://github.com/ruven/iipsrv
the server has been set following http://circle-theory.blogspot.de/2015/01/jpeg2000-support-with-iipimage-iipsrv.html although instead of
 cp ~/iipsrv/src/iipsrv.fcgi /var/www/localhost/fcgi-bin/
the file has been moved to 
 sudo cp ~/iipsrv/src/iipsrv.fcgi /usr/lib/iipimage-server.

further following 
http://iipimage.sourceforge.net/2014/12/iiif/

rewrting is set in the nginx server 
`````
 merge_slashes off;
        rewrite ^/iiif/(.*) /iipsrv/iipsrv.fcgi?IIIF=/media/add/images/%2F$1 last;
`````
so that now 
http://betamasaheft.aai.uni-hamburg.de/iiif/BMQ/003/BMQM-003_037.tif/full/full/0/default.jpg
and
http://betamasaheft.aai.uni-hamburg.de/iiif/BMQ/003/BMQM-003_037.tif/info.json

are both available for the test manuscript images

**Comment by PietroLiuzzo** [comment]
the mirador viewer has been added by following the instructions here http://projectmirador.org/docs/docs/getting-started.html
After building it I took what was into /build after the installation process and put it into my exist-db  app resources folder to be able to use it in exist. 
controller.xql had to be told to redirect internal calls to build/mirador to resources/mirador.

The viewer is loaded by constructing as a string a variable containing the needed data and adding a sjavacript with the settings calling those variables.

````xml
<script type="text/javascript">
var data = [{manifestUri: "http://digi.vatlib.it/iiif/MSS_Vat.et.1/manifest.json", location: "Biblioteca Apostolica Vaticana"}]
</script>
  <script type="text/javascript">
      $(function() {
        Mirador({
          id: "viewer",
          data: data
        });
      });
    </script>
`````
